### DISCUSSION TITLE
Mac M1 elastic search issues

### DISCUSSION THREAD

**Discussion by venkatesan-framewirk** [Sunbird-Ed/Community#426] [Bugs]
docker image of elastic search 6.8.11 isn't available in the dockerhub.
Solved using **docker pull docker.elastic.co/elasticsearch/elasticsearch:6.8.11** before running **docker run -p 9200:9200 --name sunbird_es -v $sunbird_dbs_path/es/data:/usr/share/elasticsearch/data -v $sunbird_dbs_path/es/logs://usr/share/elasticsearch/logs -v $sunbird_dbs_path/es/backups:/opt/elasticsearch/backup -e "discovery.type=single-node" --network sunbird_db_network -d docker.elastic.co/elasticsearch/elasticsearch:6.8.11**

**Comment by chitranshu-keshav** [comment]
@venkatesan-framewirk thanks for reporting the issue as well as solution. This really helps the community. We will be closing this thread
